packages/firebase/index.d.ts

```diff
@@ -1220,6 +1234,39 @@ declare namespace firebase.app {
   }
 }
 
+/**
+ * @webonly
+ */
+declare namespace firebase.installations {
+  /**
+   * The Firebase Installations service interface.
+   *
+   * Do not call this constructor directly. Instead, use
+   * {@link firebase.installations `firebase.installations()`}.
+   */
+  export interface Installations {
+    /**
+     * Creates a Firebase Installation if there isn't one for the app and
+     * returns the Installation ID.
+     *
+     * @return Firebase Installation ID
+     */
+    getId(): Promise<string>;
+
+    /**
+     * Returns an Authentication Token for the current Firebase Installation.
+     *
+     * @return Firebase Installation Authentication Token
+     */
+    getToken(): Promise<string>;
+
+    /**
+     * Deletes the Firebase Installation and all associated data.
+     */
+    delete(): Promise<void>;
```

Comment:
Why is this not called #deleteInstallation()?

Like getId above this might be clear in the context of Firebase Installations, but still why not be explicit about what is happening here?